src/selector/scrollTo.js

```diff
@@ -12,6 +12,24 @@ define([
         $target = $target || $('body');
         options = options || {};
 
+        if (options.container) {
+            var $container = $(options.container).first();
+            var container = $container[0];
+
+            // Ensure that the container is a plain DOM object, as that's what
+            // Velocity requires for non-jQuery environments
+            options.container = container;
+
+            if ($container.css('position') === 'static') {
+                $container.css('position', 'relative');
```

Comment:
Seems dangerous to change the CSS property automagically. How about only throwing an error so that the developer can change the property?